mods/ra/rules/misc.yaml

```diff
@@ -160,20 +160,33 @@ camera.paradrop:
 	RevealsShroud:
 		Range: 6c0
 
-camera.spyplane:
+detector.spyplane:
 	Inherits: CAMERA
+	-RevealsShroud:
+	-Immobile:
+	Mobile:
+		TerrainSpeeds:
+		OccupiesSpace: false
+	Passenger:
+	Parachutable:
+		KilledOnImpassableTerrain: false
+		Image:
```

Comment:
The linter is complaining because `GroundCorpseSequence` and `WaterCorpseSequence` refer to sequences on this image, which is being set to null.

Reply:
Take a look at the D2K paradrop power.  That drops units without having a visible parachute.